### PR TITLE
[PE-4575] Designed introduction page

### PR DESCRIPTION
**Pull request by mattnickles**

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/859/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://bbc6dc7afd843ff261d7c260fefa75ebc44e8a1a-chi-pr.enterprise.ctl.io" target="_blank"> https://bbc6dc7afd843ff261d7c260fefa75ebc44e8a1a-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/860/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://ee42c9e6372cbe42dd1d46969f1f48567402ef13-chi-pr.enterprise.ctl.io" target="_blank"> https://ee42c9e6372cbe42dd1d46969f1f48567402ef13-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/862/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://d9bf5a9059c319111a350e0aabc1719b3b730df8-chi-pr.enterprise.ctl.io" target="_blank"> https://d9bf5a9059c319111a350e0aabc1719b3b730df8-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/864/ for more details.

File: src/website/views/getting-started/index.pug [hunk 1]
```diff
@@ -1,12 +1,87 @@
...
+        .chi-grid.-no-gutter
+            .chi-col.-w--12.-p--2.-p-lg--0
+                h3.-text--h3.-mt--0.-mb--1.-mb-sm--2 Compatible
+                p.-text--large.-text--grey.-m--0
+                    | Chi's framework agnostic web components and independent CSS layer are compatible with any
+                    | major framework (AngularJS, ReactJS, VueJS) or no framework at all.
 
-.chi-tabs-panel#csstab
-  include ./introduction/_css
+.chi-h3.-mt--3.-d--none Getting Started
```
Comment (jllr) [inline src/website/views/getting-started/index.pug:39]: @mattnickles is this going to be released in the 2.0 prod version or are just ideas?

Comment (mattnickles) [reply src/website/views/getting-started/index.pug:39]: Just ideas. If more content was added I was going to render this at the bottom of the page to guide users where to go next.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://daaf42433c8f630e3518e36d47696c88b0523039-chi-pr.enterprise.ctl.io" target="_blank"> https://daaf42433c8f630e3518e36d47696c88b0523039-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/871/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://4e666b6d1f048adc259a092e2ea8e4a72566f350-chi-pr.enterprise.ctl.io" target="_blank"> https://4e666b6d1f048adc259a092e2ea8e4a72566f350-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/877/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://8b578d687e61d0beed1ae338c3c212b83ff8efe1-chi-pr.enterprise.ctl.io" target="_blank"> https://8b578d687e61d0beed1ae338c3c212b83ff8efe1-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/879/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-lvpmn1, https://ops.elasticbox.com/services/instances/i-2024bx, https://ops.elasticbox.com/services/instances/i-ly6d3w, https://ops.elasticbox.com/services/instances/i-vlhl1p, https://ops.elasticbox.com/services/instances/i-14x26z, https://ops.elasticbox.com/services/instances/i-gzjsod

**Pull request merged**